src/relax/analysis/udchain.cc

```diff
@@ -65,36 +65,36 @@ class UDChain : public relax::ExprVisitor {
 std::pair<runtime::Map<Var, runtime::Array<Var>>, runtime::Array<Var>> FunctionUseDef(
     const Function& fn) {
```

Comment:
The change in this file is unrelated, but I couldn't help updating this file after noticing clean-up opportunities.